tools/release_builder.py

```diff
@@ -480,7 +479,7 @@ def _update_name_and_version(self, pkgdir):
         # Update package's version to reflect the user's input
         package_json['version'] = self._pkg_version
 
-        logger.info('[1/2] Updated package.json:')
+        logger.info('[1/3] Updated package.json:')
```

Comment:
Seems almost like we could use something like a `log_progress` method where we track the current step?

Reply:
I'll just remove the `[x/y]`s as they aren't really providing anything substantive